flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -294,7 +294,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
 
         setInterval(() => {
             if (!this.socket || this.socket.readyState !== 1) {
-                this.connectWebsocket(NXDataStore.get("CONFIG_EXTERNAL_MCDU_PORT", "8080"));
+                this.connectWebsocket(NXDataStore.get("CONFIG_EXTERNAL_MCDU_PORT", "8380"));
```

Comment:
This will default to the new port for anyone who didn't change the setting in the past. 👍